support-internationalisation/src/test/scala/com/gu/i18n/CountryGroupTest.scala

```diff
@@ -1,11 +1,11 @@
 package com.gu.i18n
 
 import com.gu.i18n.Currency._
-import org.scalatest.Inspectors
-import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest._
+import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 
-class CountryGroupTest extends AsyncFlatSpec with Matchers with Inspectors {
+class CountryGroupTest extends AnyFlatSpec with Matchers with Inspectors {
```

Comment:
scala 3 scalatest noticed that these weren't actually async tests, somehow scala 2 one didn't.